net/respondd-module-airtime/src/respondd.c

```diff
@@ -22,8 +24,12 @@ static struct json_object *respondd_provider_statistics(void) {
 
 	ifaces = get_ifaces();
 	while (ifaces != NULL) {
-		get_airtime(wireless, ifaces->ifx);
-
+		ok = get_airtime(wireless, ifaces->ifx);
+		if (ok) {
+			newest_element_index = json_object_array_length(wireless)-1;
+			last = json_object_array_get_idx(wireless, newest_element_index);
+			json_object_object_add(last, "idx", json_object_new_int(ifaces->wiphy));
```

Comment:
```suggestion
			json_object_object_add(last, "wiphy", json_object_new_int(ifaces->wiphy));
```

Or maybe just `phy`? `idx` is just a bit too generic from my POV.

Reply:
+1 for `phy`

Reply:
Thats fine with me; resolved in d630d87799846a5bf2c6cc096f336360b5b660e2